Sources/SwiftGtk/enums/Align.swift

```diff
@@ -0,0 +1,61 @@
+import CGtk
+
+/// Controls how a widget deals with extra space in a single (x or y) dimension.
+/// 
+/// Alignment only matters if the widget receives a “too large” allocation, for example if you packed the widget with the `GtkWidget:expand` flag inside a `GtkBox`, then the widget might get extra space. If you have for example a 16x16 icon inside a 32x32 space, the icon could be scaled and stretched, it could be centered, or it could be positioned to one side of the space.
+/// 
+/// Note that in horizontal context `GTK_ALIGN_START` and `GTK_ALIGN_END` are interpreted relative to text direction.
+/// 
+/// `GTK_ALIGN_BASELINE` support for it is optional for containers and widgets, and it is only supported for vertical alignment. When its not supported by a child or a container it is treated as `GTK_ALIGN_FILL`.
+/// 
+/// https://docs.gtk.org/gtk3/enum.Align.html
```

Comment:
Change the URL to `See [the Gtk docs](https://docs.gtk.org/gtk3/enum.Align.html).` so that it shows up nicely in the documentation.